test/participants.js

```diff
@@ -1,166 +0,0 @@
-const assert = require("assert")
```

Comment:
this was helpful to test the user's contributions. Why has it been removed? Or has it been moved? I don't see it...